disttask/loaddata/wrapper.go

```diff
@@ -16,21 +16,28 @@ package loaddata
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/br/pkg/lightning/backend"
+	"github.com/pingcap/tidb/br/pkg/lightning/backend/encode"
+	"github.com/pingcap/tidb/br/pkg/lightning/backend/kv"
+	"github.com/pingcap/tidb/br/pkg/lightning/backend/local"
 	"github.com/pingcap/tidb/br/pkg/lightning/config"
+	"github.com/pingcap/tidb/br/pkg/lightning/log"
 	"github.com/pingcap/tidb/br/pkg/lightning/mydump"
 	"github.com/pingcap/tidb/br/pkg/storage"
+	"github.com/pingcap/tidb/ddl/ingest"
 	"github.com/pingcap/tidb/executor/importer"
+	"github.com/pingcap/tidb/parser"
+	"github.com/pingcap/tidb/parser/ast"
+	"github.com/pingcap/tidb/table/tables"
 	"github.com/pingcap/tidb/util/intest"
+	"github.com/pingcap/tidb/util/logutil"
 )
 
-func makeTableRegions(ctx context.Context, task *TaskMeta, concurrency int) ([]*mydump.TableRegion, error) {
-	if concurrency <= 0 {
-		return nil, errors.Errorf("concurrency must be greater than 0, but got %d", concurrency)
-	}
-
-	b, err := storage.ParseBackend(task.Dir, nil)
+func getStore(ctx context.Context, dir string) (storage.ExternalStorage, error) {
```

Comment:
the name "dir" indicate we only support local disk storage?

Reply:
no, also external dir

Reply:
please give a better name like storageURI